### DISCUSSION TITLE
Burbidge's "A Being"

### DISCUSSION THREAD

**Discussion by ghost** [systemphil/sphil#94] [Hegel]

**Comment by Firgrep** [comment]
Hi @Reshimu ! Thanks for opening a discussion. It looks like your question was edited out. Was this intentional?

EDIT: looking into the history of your question, you talk about Burbidge's use of indefinite article and its potential grammatical merit. 

My main concern is that the indefinite article invites notions of one and many, or even quantity, which are later categories. So it's not helpful in determining the present logic if it has to evoke later logic (which this present logic is meant to establish).

An inquiry into the meaning of articles could be beneficial if a clear demarcation is possible between language and logic.

**Comment by ghost** [comment]
Hello Filip
Yes, it was intentional, I was not happy with my reflexion… even though I
may give it another try later… of course you can delete the thread !

Le lun. 21 oct. 2024 à 13:26, Filip Niklas ***@***.***> a
écrit :

**Comment by Firgrep** [comment]
Okay, fair enough! I'll just close the thread, but you're welcome to start another one - and delete this one entirely if you like :)
